src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/stories/SettingsEditInfoNonUsUsers.stories.tsx

```diff
@@ -25,17 +25,24 @@ const meta: Meta<typeof SettingsWrapper> = {
   },
 };
 export default meta;
+
+const maxEmails = () => {
+  const emails = [];
+  for (let i = 1; i <= CONST_MAX_NUM_ADDRESSES; i++) {
+    emails.push(mockRandomEmail(i));
+  }
+
+  return emails;
+};
```

Comment:
nit: there's a nice one-liner for this instead of mutating an array:

`Array.from({ length: CONST_MAX_NUM_ADDRESSES }, (_, index) => mockRandomEmail(index));`